docs/installation/micro-rdk-dev.md

```diff
@@ -276,6 +297,8 @@ make upload
 {{% /tab %}}
 {{% /tabs %}}
 
+When prompted, select the serial port that your ESP32 is connected to through a data cable.
+
 If successful, you will retain a serial connection to the board until you press `Ctrl-C`.
 To manage this connection, consider running it within a dedicated terminal session, or under `tmux` or `screen`.
 While the serial connection is live, you can also restart the currently flashed image with `Ctrl-R`.
```

Comment:
On line 307: where do they see the last online, let's add more specificity or less. "If successful, you machine's page will show that your machine is **Live**."

Reply:
added